README.md

```diff
@@ -23,70 +22,123 @@ It provides fully customizeable image, video, and web page wallpapers that can b
+
+```bash
+sudo dnf install meson vala gcc-c++ gtk3-devel clutter-devel clutter-gtk-devel clutter-gst3-devel webkit2gtk3-devel libgee-devel gstreamer1-libav
 ```
-sudo add-apt-repository ppa:gnome3-team/gnome3 -y
-sudo add-apt-repository ppa:vala-team -y
-sudo add-apt-repository ppa:gnome3-team/gnome3-staging -y
-sudo apt install meson valac libgtk-3-dev libgee-0.8-dev libclutter-gtk-1.0-dev libclutter-1.0-dev libwebkit2gtk-4.0-dev libclutter-gst-3.0-dev
+
+There is no `.rpm` package yet, jump to the [compilling section](#compilling); if you can package Komorebi for Fedora, go ahead and let us know!
+
+### Arch Linux and derivates (Manjaro, etc...)
+
+Install from the [AUR](https://aur.archlinux.org/packages/komorebi/):
+
+```bash
+sudo yay -S komorebi
+```
+
+or grab the required dependencies:
+
+```bash
+sudo pacman -S meson vala gtk3 clutter clutter-gtk clutter-gst libgee
+```
+
+and jump to the [compilling section](#compilling).
+
+## Compilling
+
+Run the following:
+```bash
 git clone https://github.com/Komorebi-Fork/komorebi.git
 cd komorebi
-meson build && cd build && meson install
+meson build && cd build && ninja
 ```
 
-## Change Wallpaper & Desktop Preferences
+To install it as well, run this after the above:
+```bash
+meson install
+```
+
+## Using Komorebi
+
+Simply run `komorebi`, or open your apps and look for **Komorebi**.
+
+Optional args:
+
+- `--single-screen`: forces Komorebi to run on the main screen only
+- `version` or `--version`: prints current version
+
+### Change wallpaper & desktop preferences
 To change desktop preferences or your wallpaper, right click anywhere on the desktop to show the menu.
 
 ![s1](https://raw.githubusercontent.com/Komorebi-Fork/komorebi/master/screenshots/preferences.jpg)
 
-## How do I create my own wallpaper?
+### Create custom wallpapers
 
-Komorebi provides a simple tool to create your own wallpapers! Simply, open your apps and search for 'Wallpaper Creator'
+Komorebi provides a simple tool to create your own wallpapers! Simply run `komorebi-wallpaper-creator` or open your apps and search for **Wallpaper Creator**.
 
 ![s1](https://raw.githubusercontent.com/Komorebi-Fork/komorebi/master/screenshots/wallpaper_creator.jpg)
 
 You can use either an image, a video, or a web page as a wallpaper and you have many different options to customize your very own wallpaper!
 
-## Uninstall
+## Uninstalling
 
-### If you installed a packaged version of Komorebi
+If you manually installed Komorebi, run the following on the cloned repository folder:
 
-1. Open Terminal
-2. `sudo apt remove komorebi`
-
-### If you manually installed Komorebi
+```bash
+cd build
+sudo ninja uninstall
+```
 
-1. Open Terminal
-2. `cd komorebi/build`
-3. `sudo ninja uninstall`
+If you didn't compile Komorebi and install it from a pre-combiled package, uninstall it through your package manager like any other package.
```

Comment:
If you didn't compile Komorebi from source you can uninstall it through your package manager like any other package.